modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc

```diff
@@ -28,14 +28,14 @@ $ oc create namespace httpbin
 $ oc label namespace httpbin istio-injection=enabled
 ----
 
-. Deploy a sample service by applying the `http.yaml` file: 
+. Deploy a sample service named `httpbin` by applying the `httpbin.yaml` file: 
```

Comment:
This also needs that command language, but already has "by [doing some action]" wording. I'll leave it up to you to figure out, but maybe this could work:
```suggestion
. Apply the `httpbin.yaml` file to deploy a sample service named `httpbin` by running the following command: 
```
Not sure I'm satisfied with that, but you get the point and I trust that you know what's right. Maybe one of the "apply the yaml file" or "deploy a service" phrases can be dropped and still have the step be reasonably informative. 

Or maybe you can say "this deploys a service named X" at the end, even though we generally should avoid describing the results of steps like that

Reply:
done